oak/server/loader/oak_runner_main.cc

```diff
@@ -53,6 +53,11 @@ std::shared_ptr<grpc::ServerCredentials> BuildTlsCredentials(std::string pem_roo
 int main(int argc, char* argv[]) {
   absl::ParseCommandLine(argc, argv);
 
+#ifdef OAK_DEBUG
+  // Use the failure signal handler from glog.
```

Comment:
Could you add a description of what this actually does? AFAICT it prints a stack trace upon receiving various signals.